casper/src/main/scala/coop/rchain/casper/storage/RNodeKeyValueStoreManager.scala

```diff
@@ -62,7 +61,6 @@ object RNodeKeyValueStoreManager {
       (Db("eval-roots"), evalHistoryEnvConfig),
       (Db("eval-cold"), evalColdEnvConfig),
       // RSpace cache (used for block merge)
-      (Db("rspace-channels"), rspaceCacheEnvConfig),
```

Comment:
this is duplicate for `channels` and not used

Reply:
Do we have this fixed on some other place? Channels map DB should not be shared with evaluator.

Channels map DB in Casper RSpace should be in the `rspace` folder with other DBs and for evaluator we can use dummy DB to not create any mark on disk.

Reply:
Changes in PR #3367 (https://github.com/rchain/rchain/pull/3367/commits/fe7821a3bab32abea7ff361c312acd3582209a84) should cover this fix so not needed here.